src/openvic-simulation/dataloader/Dataloader.cpp

```diff
@@ -150,13 +152,19 @@ bool Dataloader::_load_pop_types(PopManager& pop_manager, fs::path const& pop_ty
 			return pop_manager.load_pop_type_file(file.stem().string(), _parse_defines(file).get_file_node());
 		}
 	);
-	if (!ret) {
-		Logger::error("Failed to load pop types!");
-	}
```

Comment:
Why remove this in a commit about military stuff?

Reply:
Just something I noticed - that was being printed twice. And I've been told to avoid making commits for these small things, so I just added it since I was editing that file anyway.